src/app/content/highlights/components/EditCard.tsx

```diff
@@ -61,7 +68,8 @@ const EditCard = React.forwardRef<HTMLElement, Props>((
 
   React.useEffect(onClickOutside(element, isFocused, blurIfNotEditing), [isFocused, editingAnnotation]);
 
-  const onColorChange = (color: HighlightColorEnum) => {
+  const onColorChange = (color: HighlightColorEnum, flag?: boolean) => {
+    services.analytics.createNote.track(book, flag ? 'default' : color);
```

Comment:
if you look at some other events you'll see sometimes i put a selector in the event definition to make it easy to collect the data the event expects, this way if we decide we need more data we only have to change it in one place in the event definition. you can then access it here and pass it through. something like: 

```
import {useSelector} from 'react-redux';

...in the component somewhere
const noteEventData = useSelector(services.analytics.createNote.selector);
services.analytics.createNote.track(noteEventData, userSelected ? color : 'default');

```